Cineaste/Extension/Int16+Formatter.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Int16+Formatter.swift
+//  Cineaste
+//
+//  Created by Felizia Bernutz on 14.04.18.
+//  Copyright © 2018 notimeforthat.org. All rights reserved.
+//
+
+import Foundation
+
+extension Int16 {
+    private static let numberFormatter = { () -> NumberFormatter in
+        let formatter = NumberFormatter()
+        formatter.numberStyle = .none
+        return formatter
+    }()
+
+    var formattedForRuntime: String {
```

Comment:
As we have a property for formatted vote average for a movie i would also move this to the movie entity.

Reply:
That's a good advice. I'll fix that.